app/src/main/java/org/sopt/tabling/presentation/queue/ReservationDoneViewHolder.kt

```diff
@@ -0,0 +1,15 @@
+package org.sopt.tabling.presentation.queue
+
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.tabling.databinding.ReservationItemDoneBinding
+import org.sopt.tabling.domain.model.ReservationItem
+
+class ReservationDoneViewHolder(private val binding: ReservationItemDoneBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+    fun onBind(item: ReservationItem.ReservationDoneView) {
+        binding.tvReservationDate.text = item.reserveDate
+        binding.tvReservationPeople.text = item.reserveNum + "명"
```

Comment:
요런 것도 스트링 추출 해서 사용할 수 있어용
string 리소스에 %d명 이런 식으로 선언해주고 
```suggestion
        binding.tvReservationPeople.text = context.getString(R.reserve_num, item.reserveNum)
```
얘처럼 하면 됨요 !

Reply:
데이터바인딩 안 쓰면 안 되는 군여...............ㅜㅜ 하아 데이터바인딩 공부도 얼른 해야되는데ㅜㅜㅜ 감사합니다  조은 방법